include/sdf/Root.hh

```diff
@@ -58,6 +59,17 @@ namespace sdf
     /// \brief Default constructor
     public: Root();
 
+    /// \brief Get the name of the world without loading the entire world
+    /// Users shouldn't normally need to use this API.
+    /// This doesn't load the world, it might return the world name even if the
+    /// file is not a valid SDFormat file.
+    /// \param[in] _filename Name of the SDF file to parse.
+    /// \param[out] _worldNames A vector with world names
+    /// \return Errors, which is a vector of Error objects. Each Error includes
+    /// an error code and message. An empty vector indicates no error.
+    public: Errors WorldNameFromFile(const std::string &_filename,
```

Comment:
nit: should this be `WorldNamesFromFile` (plural)?

Reply:
https://github.com/gazebosim/sdformat/pull/1027/commits/a6c9ef3e32dc5500fbb57c1b55c7247e021baf51